### PR TITLE
Improved error handling for Domo update

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -63,8 +63,12 @@ def mark_users_updated
 
     def update_domo
       dscd = DailyStudentCountToDomo.new
-      dscd.call
-      flash[:notice] = "The update completed."
+      successful = dscd.call
```
Comment (BryanEliDimas) [inline app/controllers/admin/users_controller.rb:66]: I didn't notice this in my previous code review, but this naming style is non-ruby-esque by the way. I'd recommend something like send_daily_count.

Comment (edwoodward) [reply app/controllers/admin/users_controller.rb:66]: @BryanEliDimas Are you talking about the function name or the variable name?

Comment (BryanEliDimas) [reply app/controllers/admin/users_controller.rb:66]: Variable

Comment (BryanEliDimas) [reply app/controllers/admin/users_controller.rb:66]: No big deal

Comment (edwoodward) [reply app/controllers/admin/users_controller.rb]: @BryanEliDimas Variable updated

**Pull request merged**